docs/06-actors-actions.md

```diff
@@ -0,0 +1,414 @@
+player.vel.x = 5
+
+// add player to the current scene
+game.add(player)
+```
+
+[[Engine.add|game.add]] is a convenience method for adding an actor to the current scene. The equivalent verbose call is [[Scene.add|game.currentScene.add]].
+
+## Actor Lifecycle
+
+An actor has a basic lifecycle that dictates how it is initialized, updated, and drawn. Once an actor is part of a
+[scene](/docs/scene), it will follow this lifecycle.
+
+![Actor Lifecycle](/assets/images/docs/ActorLifecycle.png)
+
+## Extending actors
+
+For "real-world" games, you'll want to extend the `Actor` class.
+This gives you much greater control and encapsulates logic for that
+actor.
+
+You can override the [[Actor.onInitialize]] method to perform any startup logic
+for an actor (such as configuring state). `onInitialize` gets called
+**once** before the first frame an actor is drawn/updated. It is passed
+an instance of [Engine](/docs/intro) to access global state or perform coordinate math.
+
+```ts
+class Player extends ex.Actor {
+  public level = 1
+  public endurance = 0
+  public fortitude = 0
+
+  constructor() {
+    super({ x: 50, y: 50 })
+  }
+
+  public onInitialize(engine: ex.Engine) {
+    this.endurance = 20
+    this.fortitude = 16
+  }
+
+  public getMaxHealth() {
+    return 0.4 * this.endurance + 0.9 * this.fortitude + this.level * 1.2
+  }
+}
+```
+
+## Updating actors
+
+In most games, things are happening on screen: the background is parallaxing, your hero responds to input, or enemies shoot bullets. In Excalibur, the logic that updates game state is run during the [update loop](/docs/intro#engine-lifecycle). Actors are a way to encapsulate that logic, such as a `Player` or `Enemy` or `MenuButton`. Actors can be pretty much anything!
+
+### Managing game state
+
+Excalibur does not provide any out-of-the-box way to manage game state but typically you can either use class properties or introduce something more sophisticated like a [state machine](https://github.com/davidkpiano/xstate).
+
+The benefit of something like a state machine is that state can be separated from the actions an actor may take and you can then _save_ and _load_ state more easily to enable save game management. You could choose for example to have a global game state that you can serialize and deserialize.
+
+<docs-note>Have you implemented state management in your Excalibur game? [Let us know](https://github.com/excaliburjs/Excalibur#questions)!</docs-note>
+
+## Update hooks
+
+There are three ways to hook into the update loop of an actor: [[Actor.onPreUpdate]], [[Actor.update]] and [[Actor.onPostUpdate]]. Actors (and other entities in Excalibur) all have "core" logic that runs in the update or draw loop. The pre- and post-method hooks allow you to choose when you want to run logic in each phase. _Normally_ you will run logic in the "pre" hook but sometimes you may want to completely override the core logic or run logic that uses state that was updated _after_ the core logic runs.
```

Comment:
I would be clear that overriding the `update()` removes excaliburs core logic. And usually peeps want `onPostUpdate`